ern-composite-gen/src/createMetroConfig.ts

```diff
@@ -2,23 +2,29 @@ import fs from 'fs-extra';
 import path from 'path';
 import beautify from 'js-beautify';
 import os from 'os';
+import semver from 'semver';
 
 export async function createMetroConfig({
   cwd,
   projectRoot,
   blacklistRe,
   extraNodeModules,
   watchFolders,
+  reactNativeVersion,
 }: {
   cwd?: string;
   projectRoot?: string;
   blacklistRe?: RegExp[];
   extraNodeModules?: { [pkg: string]: string };
   watchFolders?: string[];
+  reactNativeVersion: string;
 }) {
+  const blackistSource = semver.gte(reactNativeVersion, '0.64.0')
```

Comment:
50/50 on utility method, since this is a single statement right now. Maybe in the future if this ever becomes more complex, we can consider moving is somewhere else?

Also:

```suggestion
  const blacklistSource = semver.gte(reactNativeVersion, '0.64.0')
```